product_variants_no_automatic_creation/models/product_configurator.py

```diff
@@ -31,8 +31,10 @@ class ProductConfigurator(models.AbstractModel):
     @api.depends('product_attribute_ids', 'product_attribute_ids.value_id')
     def _compute_price_extra(self):
         for record in self:
-            record.price_extra = sum(
-                record.mapped('product_attribute_ids.price_extra'))
+            for variant_id in record.attribute_value_ids:
```

Comment:
Esto debe hacerse sólo cuando product_id no está establecido. Además, utiliza `mapped` y `filtered` en una sola línea que queda mucho más elegante y legible, en lugar de tanto for.